Firmware/RTK_Surveyor/RTK_Surveyor.ino

```diff
@@ -32,6 +32,9 @@
 #define COMPILE_L_BAND //Comment out to remove L-Band functionality
 #define COMPILE_SD_MMC // Comment out to remove REFERENCE_STATION microSD SD_MMC support
 #define ENABLE_DEVELOPER //Uncomment this line to enable special developer modes (don't check power button at startup)
+//#define FACTORY_RESET_AT_POWER_ON //Uncomment this line to perform a factory reset at every power-on. Needs ENABLE_DEVELOPER
```

Comment:
This block is meant for code size reduction, not really control (I know, ENABLE_DEVELOPER...). Do you need these at boot? I believe they (or nearly the same functionality) are built into the menuSystem and menuDebug menus. That's how I erase LFS and run for start-from-scratch tests. Is there a good reason not to build these into menus and remove from this define block?

Reply:
Again, I was in the weeds with my corrupt settings - in both LittleFS and SD_MMC. (The settings were clean initially, but then got corrupted and those corrupt SD_MMC settings were being copied into LittleFS. All due to that `SD_MMC.end()`). We can safely delete these now.